tools/api-generator/overlays/groupID_flag.yaml

```diff
@@ -0,0 +1,10 @@
+overlay: 1.0.0
+info:
+  title: Configure groupId flag
+  version: 1.0.0
+actions:
+  - target: $.components.parameters.groupId
+    update:
+      x-xgen-atlascli:
+        aliases: 
+          - projectId
```

Comment:
nit, it works the way it is, but currently, we have a small gap we can't provide the short version of the alias (we only provide --projectId but we don't have a way to provide a hypothetical -P)

Reply:
Good suggestion, I'll make the changes!

Reply:
Updated